L1Trigger/Phase2L1Taus/interface/HPSPFTauBuilder.h

```diff
@@ -1,20 +1,20 @@
-#ifndef L1Trigger_Phase2L1Taus_L1HPSPFTauBuilder_h
-#define L1Trigger_Phase2L1Taus_L1HPSPFTauBuilder_h
+#ifndef L1Trigger_Phase2L1Taus_HPSPFTauBuilder_h
+#define L1Trigger_Phase2L1Taus_HPSPFTauBuilder_h
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"             // edm::ParameterSet
 #include "DataFormats/Provenance/interface/ProductID.h"             // edm::ProductID
-#include "L1Trigger/Phase2L1Taus/interface/L1HPSPFTauQualityCut.h"  // L1HPSPFTauQualityCut
-#include "DataFormats/Phase2L1Taus/interface/L1HPSPFTau.h"          // l1t::L1HPSPFTau
+#include "L1Trigger/Phase2L1Taus/interface/HPSPFTauQualityCut.h"  // HPSPFTauQualityCut
+#include "DataFormats/L1TParticleFlow/interface/HPSPFTau.h"          // l1t::HPSPFTau
 #include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"  // l1t::PFCandidate, l1t::PFCandidateCollection, l1t::PFCandidateRef
 #include "DataFormats/JetReco/interface/CaloJet.h"
 #include "DataFormats/L1TCorrelator/interface/TkPrimaryVertex.h"
 #include "CommonTools/Utils/interface/FormulaEvaluator.h"
 #include <vector>
 
-class L1HPSPFTauBuilder {
+class HPSPFTauBuilder {
```

Comment:
I think you should keep the original name `L1HPSPFTauBuilder`, **or** move the class into the `l1t` namespace

Reply:
Done